bot/cogs/dm_relay.py

```diff
@@ -17,30 +19,45 @@
 class DMRelay(Cog):
     """Relay direct messages to and from the bot."""
 
+    # RedisCache[str, t.Union[discord.User.id, discord.Member.id]]
+    dm_cache = RedisCache()
+
     def __init__(self, bot: Bot):
         self.bot = bot
         self.webhook_id = constants.Webhooks.dm_log
         self.webhook = None
         self.bot.loop.create_task(self.fetch_webhook())
 
     @commands.command(aliases=("reply",))
-    async def send_dm(self, ctx: commands.Context, member: discord.Member, *, message: str) -> None:
+    async def send_dm(self, ctx: commands.Context, member: Optional[discord.Member], *, message: str) -> None:
```

Comment:
This converter will convert by name and nickname too. So if someone happens to have a common word as a name, you'll end up DMing them.

Reply:
That's a very good point. What I really want is a converter that only does mentions and userIDs. Maybe I'll need to make a custom one?

Reply:
Fixed in 403572b